.github/workflows/gradle-docker-ci.yml

```diff
@@ -0,0 +1,34 @@
+##
+# Created by brightSPARK Labs
+# www.brightsparklabs.com
+##
+
+name: Gradle Docker CI
+
+on:
+  push:
+    branches: [ feature, develop, master ]
+  pull_request:
+    branches: [ feature, develop, master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'temurin'
+    - name: Run Unit Tests
+      run: ./gradlew build
+    - name: Publish
+      if: github.ref == 'refs/heads/master'
```

Comment:
I'd kind of prefer if we did test and publish in separate workflows so it's more clear what's happening.